src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/Content/Controls/TextBoxSamplePage.xaml

```diff
@@ -102,6 +102,86 @@
 					</StackPanel>
 				</DataTemplate>
 			</sample:SamplePageLayout.MaterialTemplate>
+			<sample:SamplePageLayout.M3MaterialTemplate>
+				<DataTemplate>
+					<StackPanel>
+						<!-- TextBox Filled -->
+						<smtx:XamlDisplay UniqueKey="M3Material_TextBoxSamplePage_1"
+										  Style="{StaticResource XamlDisplayBelowStyle}">
+							<TextBox Style="{StaticResource M3MaterialFilledTextBoxStyle}" />
+						</smtx:XamlDisplay>
+
+						<!-- TextBox Filled With PlaceholderText -->
+						<smtx:XamlDisplay UniqueKey="M3Material_TextBoxSamplePage_2"
+										  Style="{StaticResource XamlDisplayBelowStyle}">
+							<TextBox PlaceholderText="Filled"
+									 Style="{StaticResource M3MaterialFilledTextBoxStyle}" />
+						</smtx:XamlDisplay>
+
+						<!-- TextBox Filled Disabled with PlaceholderText -->
+						<smtx:XamlDisplay UniqueKey="M3Material_TextBoxSamplePage_3"
+										  Style="{StaticResource XamlDisplayBelowStyle}">
+							<TextBox PlaceholderText="Placeholder"
+									 Text="Filled Disabled"
+									 Style="{StaticResource M3MaterialFilledTextBoxStyle}"
+									 IsEnabled="False" />
+						</smtx:XamlDisplay>
+
+						<!-- TextBox Outlined -->
+						<!--<smtx:XamlDisplay UniqueKey="M3Material_TextBoxSamplePage_4"
+										  Style="{StaticResource XamlDisplayBelowStyle}">
+							<TextBox Style="{StaticResource M3MaterialOutlinedTextBoxStyle}" />
+						</smtx:XamlDisplay>-->
+
+						<!-- TextBox Outlined with PlaceholderText -->
+						<!--<smtx:XamlDisplay UniqueKey="M3Material_TextBoxSamplePage_5"
+										  Style="{StaticResource XamlDisplayBelowStyle}">
+							<TextBox PlaceholderText="Outlined"
+									 Style="{StaticResource M3MaterialOutlinedTextBoxStyle}" />
```

Comment:
@kazo0 did you find what's not working for the M3MaterialOutlinedTextBoxStyle for UWP at the end ?
(as these samples are still commented)